doc/lsst.ctrl.bps.htcondor/userguide.rst

```diff
@@ -321,6 +321,84 @@ existing file.
+
+.. code-block::
+
+   condor_q -held
+
+If any of your jobs are being held, it will display something similar to::
+
+    -- Schedd: sdfrome002.sdf.slac.stanford.edu : <172.24.33.226:21305?... @ 10/02/24 10:59:41
+    ID           OWNER  HELD_SINCE  HOLD_REASON
+    5471792.0    jdoe   7/10 08:27  File '/sdf/group/rubin/sw/conda/envs/lsst-scipipe-8.0.0/bin/condor_dagman' is missing or not executable
+    7636239.0    jdoe   3/20 01:32  Job raised a signal 11. Handling signal as if job has gone over memory limit.
+    5497548.0    jdoe   3/6  00:14  Job raised a signal 9. Handling signal as if job has gone over memory limit.
+
+.. note::
+
+   If you would like to display held jobs that were submitted for execution
+   by other users, use ``condor_q -held <username>`` instead where
+   ``<username>`` is the user account which held jobs you would like to check.
+   See `condor_q`_ man page for other supported options.
+
+The job that is in the hold state can be released from it with
+`condor_release`_ providing the issue that made HTCondor put it in this state
+has been resolved. For example, if your job with id 1234.0 was placed in the
+hold state because during the execution it exceeded 2048 MiB you requested for
+it during the submission, you can double the amount of memory it should request with
+
+.. code-block::
+
+   condor_qedit 1234.0 RequestMemory=4096
+
+and than release it from the hold state with
+
+.. code-block::
+
+   condor_release 1234.0
+
+If multiple jobs were placed by HTCondor in the hold state for the same reason
+(i.e. out of memory error) and the issue which caused that can be fixed in the
+same manner, you can use your username instead of job ids to handle them all
+at once:
+
+.. code-block::
+
+   condor_qedit jdoe RequestMemory=4096
+   condor_release jdoe
+
+If you only want to deal with a subset of currently held jobs, use
+``-constraint <expression>`` option that both `condor_qedit`_ and
+`condor_release`_ support where ``<expression>`` can be an arbitrarily complex
+`HTCondor ClassAd`__ expression.  For example
+
+.. code-block::
+
+   condor_qedit -constraint "JobStatus == 5 && HoldReasonCode == 3 && HoldReasonSubCode == 34" RequestMemory=4096
+   condor_release -constraint "JobStatus == 5 && HoldReasonCode == 3 && HoldReasonSubCode == 34"
+
+will only affect jobs that were placed in the hold state (``JobStatus`` is 5)
+for a specific reason, here, the memory usage exceeded memory limits
+(``HoldReasonCode`` is 3 *and* ``HoldReasonSubCode`` is 34).
+
+.. __: https://htcondor.readthedocs.io/en/latest/classads/index.html
+
+.. note::
+
+   If your jobs are being placed in the hold state often due to out of memory
+   errors, consider using automatic memory scaling that this plugin supports.
+   See `Automatic memory scaling`__ section in **ctrl_bps** documentation.
+
+.. __: https://pipelines.lsst.io/v/weekly/modules/lsst.ctrl.bps/quickstart.html#automatic-memory-scaling
```

Comment:
Using automatic memory scaling is the current default, isn't it?  If this is the case, I would mention that at the beginning and say that the examples should occur only if explicitly turned it off in the submit yaml.